geometry_msgs/Point point
int16 map_id